src/soroban_rpc.ts

```diff
@@ -127,7 +126,7 @@ export namespace SorobanRpc {
 
   export interface SimulateTransactionResponse {
     id: string;
-    error?: jsonrpc.Error;
+    error?: string;
```

Comment:
ah, we don't have much in tests that exercise the ts bindings.

Reply:
nope, no "integration" tests unfortunately